dwave_networkx/generators/chimera.py

```diff
@@ -252,6 +254,18 @@ def checkadd(v, q):
     return G
 
 
+@CHIMERA.defect_free_graph.implementation
+def defect_free_chimera(G):
+    """Construct a defect-free Chimera graph based on the properties of G."""
+    attrib = G.graph
+    family = attrib.get('family')
+    if family != CHIMERA:
```

Comment:
```suggestion
    if family is not CHIMERA:
```
Similarly in other files.